octoprint_mrbeam/translations/es/LC_MESSAGES/messages.po

```diff
@@ -4926,11 +4933,15 @@ msgstr ""
 msgid ""
 "We understand that your privacy is crucial, and we want to assure you that "
 "none of the data collected includes your personal information such as your "
-"name, login details, designs, or any other private content."
+"name, login details, designs, or any other private content. You are in full "
+"control, and you can change your mind at any time by going to "
+"<strong>\"Settings > Better together\"</strong>."
 msgstr ""
 "Entendemos que tu privacidad es crucial, y queremos asegurarte que ninguno "
 "de los datos recogidos incluye tu información personal, como tu nombre, "
-"datos de acceso, diseños o cualquier otro contenido privado."
+"datos de acceso, diseños o cualquier otro contenido privado. Tú tienes todo "
+"el control, y puedes cambiar de opinión en cualquier momento yendo a "
+"<strong>“Ajustes > Mejor juntos”</strong>."
```

Comment:
Oops, sorry... Just noticed that this wasn't changed after discussing with Cristina.
```suggestion
"<strong>“Ajustes > Mejorando juntos”</strong>."
```